package-lock.json

```diff
@@ -310,14 +310,14 @@
       "dev": true
     },
     "aws-sdk": {
-      "version": "2.443.0",
-      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.443.0.tgz",
-      "integrity": "sha512-I4vjwridWEQDq6L6GE3AsV0MzvRI1drPbHtxbI/K4Q4lmwMkskue+qiSlac4EY3ZJBuBIXGgIclKnkr8AuW71g==",
+      "version": "2.1104.0",
```

Comment:
Although it's not a major version, it's a big upgrade of seven minor versions, so it's worth performing a thorough sanity check